recipe/meta.yaml

```diff
@@ -4,6 +4,8 @@
 {% set version = "23.3" %}
 # use run or skip
 {% set unit_tests = "skip" %}
+{% set build = 7 %}
+{% set build = build + 100 %}  # [mpi != 'nompi']
```

Comment:
What is the reason to have a different build number for mpi/nompi?

Reply:
Soery it should be mpi=='nompi'. Per https://github.com/conda-forge/ambertools-feedstock/issues/132#issuecomment-1809073915 users may prefer nompi

Reply:
I mean, why do the `nompi` and `mpi` have/need different build numbers? Is that convention for `mpi` enabled/aware packages on conda-forge? 

You also might want to try and get this working on the linux builds first and skip the other builds to save you some time getting things to pass on the CI. Working locally might be a good idea as well `python build-locally.py` since it will be faster than the azure CI. Thanks for taking this on!

Reply:
> Is that convention for mpi enabled/aware packages on conda-forge?

It is explained in the documentation: https://conda-forge.org/docs/maintainer/knowledge_base.html

> Up to here, mpi providers have no explicit preference. When choosing an MPI provider, the mutual exclusivity of the mpi metapackage allows picking between mpi providers by installing an mpi provider, e.g.
>
> conda install mpich ptscotch
or
>
> conda install openmpi ptscotch
This doesn’t extend to nompi, because there is no nompi variant of the mpi metapackage. And there probably shouldn’t be, because some packages built with mpi don’t preclude other packages in the env that may have an mpi variant from using the no-mpi variant of the library (e.g. for a long time, fenics used mpi with no-mpi hdf5 since there was no parallel hdf5 yet. This works fine, though some features may not be available).
>
> Typically, if there is a preference it will be for the serial build, such that installers/requirers of the package only get the mpi build if explicitly requested. We use a higher build number for the nompi variant in this case.

Reply:
Using build numbers to trick solvers into preferring particular variants is unfortunately pervasive. I introduced it into GROMACS as a way of keeping single precision the default when I added double precision builds. I'd be nice to use a better solution if one exists - the documentation is correct but could be out of date - I haven't looked into it lately

Reply:
Sounds good! I am fine keeping `nompi` the default flavor

Reply:
```suggestion
{% set build = build + 100 %}  # [mpi == 'nompi']
```